inc/autoload.function.php

```diff
@@ -287,8 +287,17 @@ function glpi_autoload($classname) {
 
    if ($plug = isPluginItemType($classname)) {
       $plugname = strtolower($plug['plugin']);
-      $dir      = GLPI_ROOT . "/plugins/$plugname/inc/";
       $item     = str_replace('\\', '/', strtolower($plug['class']));
+
+      //try to load from marketplace folder
+      $dir = GLPI_MARKETPLACE_DIR . "/$plugname/inc/";
+
+      //then load from former plugins folder
+      if (!is_dir($dir)) {
+         $dir = GLPI_ROOT . "/plugins/$plugname/inc/";
+      }
```

Comment:
IMHO, plugins directory (proper installtion) should have higher priority than maketplace one.

Reply:
Can be done by redefining `PLUGINS_DIRECTORIES`.

Reply:
No, default values have to be sane !
So order have to be change

Reply:
Order has been fixed.